WooCommerce/Classes/ViewRelated/Reviews/ReviewsViewController.swift

```diff
@@ -122,7 +123,9 @@ final class ReviewsViewController: UIViewController {
         resetApplicationBadge()
         transitionToResultsUpdatedState()
 
-        syncingCoordinator.synchronizeFirstPage()
+        if isEmpty {
+            transitionToSyncingState(pageNumber: SyncingCoordinator.Defaults.pageFirstIndex)
```

Comment:
I think there is a bug in updating the UI only from the `SyncingCoordinator` because some cases are not managed. For example, we don't want to reload all the things when the Reviews screen appears, so it's not useful to resynchronize the SyncingCoordinator in `viewWillAppear`, and without the `transitionToSyncingState` here when the screen is empty, you will see the loading/ghost cells with the data of the old store. It can be caused by the wrong reuse of the cells but should be fixed when we will drop the old `SyncingCoordinator`. I also noticed that we are using `transitionToResultsUpdatedState` here, but in theory, also it should be called only inside the `SyncingCoordinator` delegates methods.

Reply:
I see, yea we can remove the manual UI state transitions after replacing `SyncingCoordinator` 👍 this might also not be necessary once a `ReviewsViewController` is tied to one site ID